CampaignConfig/mdc2020_corsika.ini

```diff
@@ -6,14 +6,14 @@ campaign_stage_list = generation_par, generation, resampler_par, resampler, spil
 
 [campaign_defaults]
 vo_role=Production
-campaign_keywords={"s1type": "All", "testid": ""}
+campaign_keywords={"s1type": "All", "testid": "prod", "desc": "MDC2020ae"}
```

Comment:
I thought testid was supposed to be null for production.  Will this value 'prod' propagate into collection names?

Reply:
POMS does not like null. 'prod' will not propagate into collection names.